apps/social/src/components/on-login/OnLogin.tsx

```diff
@@ -0,0 +1,16 @@
+import { useAccount } from '@gear-js/react-hooks';
+import { ReactNode } from 'react';
+
+type Props = {
+  children: ReactNode;
+  fallback?: ReactNode;
+};
+
+function OnLogin({ children, fallback }: Props) {
+  const { account } = useAccount();
+
+  // eslint-disable-next-line react/jsx-no-useless-fragment
+  return <>{account ? children : fallback}</>;
```

Comment:
`return account ? children : fallback};`

Reply:
everything is correct here. as it should return in JSX wrapper

Reply:
skipped this issue for a moment